locksmith/__tests__/app.test.ts

```diff
@@ -1,7 +1,7 @@
-const request = require('supertest')
-const app = require('../src/app')
-const Lock = require('../src/lock')
-const Transaction = require('../src/transaction')
+import request = require('supertest')
```

Comment:
TypeScript was fairly keen on having me change from `const x = require...` to `import x = require...`. 

It seems that is the current best practice for importing non-ES6 modules: https://stackoverflow.com/questions/29596714/new-es6-syntax-for-importing-commonjs-amd-modules-i-e-import-foo-require

Reply:
what happens if you use the regular `import request from 'supertest'`?